src/capsule/BiometricSessionManager.ts

```diff
@@ -0,0 +1,42 @@
+import userManagementClient from './UserManagementClient'
+import { ChallengeStorage } from './ChallengeStorage'
+
+export default class BiometricSessionManager {
+  private userId: string
+  private biometricStorage: ChallengeStorage
+  public async setBiometrics() {
+    return await userManagementClient.addBiometrics(this.userId, {
+      publicKey: await this.biometricStorage.getPublicKey(),
```

Comment:
nit: we should do a `this.biometricStorage` null check first just to return a well defined error